DiscordChatExporter.Cli/Commands/Base/ExportCommandBase.cs

```diff
@@ -252,26 +256,33 @@ await console.Output.WriteLineAsync(
             );
         }
 
-        // Print errors
-        if (errorsByChannel.Any())
+        // Print errors and warnings
+        if (errorsByChannel.Any() || warningsByChannel.Any())
         {
-            await console.Output.WriteLineAsync();
-
-            using (console.WithForegroundColor(ConsoleColor.Red))
+            var tuples = new List<(ConcurrentDictionary<Channel, string>, ConsoleColor, string)>
             {
-                await console.Error.WriteLineAsync(
-                    $"Failed to export {errorsByChannel.Count} the following channel(s):"
-                );
-            }
+                (errorsByChannel,   ConsoleColor.Red,    "Failed to export the following channels:"),
+                (warningsByChannel, ConsoleColor.Yellow, "No messages exported for the following channels:")
+            };
```

Comment:
I'd rather split these into two similar code blocks instead (one for errors, one for warnings), for readability